test/test_torch.py

```diff
@@ -4117,9 +4117,18 @@ def __reduce__(self):
                 self.assertEqual(x.size(), load_x.size())
                 self.assertEqual(x.stride(), load_x.stride())
 
+
+    def test_serialization_save_warnings(self):
+        with warnings.catch_warnings(record=True) as warns:
+            with tempfile.NamedTemporaryFile() as checkpoint:
+                x = torch.save(torch.nn.Module(), checkpoint)
+                self.assertEquals(len(warns), 0)
```

Comment:
Does this actually work? I thought that you have to check `len(warns)` outside the context manager scope.

Does the test fail without your changes? If not, then the test won't help catch a future regression

Reply:
Okay, I read up on the docs and this should work. Please ignore my comment